.github/workflows/preview_bump_and_publish.yml

```
@@ -13,18 +13,20 @@ jobs:
       - name: Checkout 🛎️
         uses: actions/checkout@v4
         with:
-          persist-credentials: false
-
+          token: ${{ secrets.MARSHMALLOW_CI_PAT }}
       - name: Setup Node 👾
         uses: actions/setup-node@v4
         with:
           node-version-file: '.nvmrc'
-
       - name: Install Dependencies 🔧
         run: |
           npm ci
+      - name: Configure Git
+        run: |
+          git config user.name "marshmallow-ci"
+          git config user.email "ci@marshmallow-insurance.com"
```

need to replace with the actual marshmallow-ci email
```suggestion
          git config user.email "ci@marshmallow-insurance.com"
```